tasks/main.yml

```diff
@@ -10,36 +10,92 @@
     #update_cache: yes
   when: ansible_distribution_release=='jessie'
 
-- name: Update repo cache
+- name: Set facts depending on OS family (Debian)
+  set_fact:
+    etc_freeradius: /etc/freeradius
```

Comment:
In future /etc/freeradius/3.0 at least for Stretch & Buster.

Reply:
and on line 103 revert to /etc/freeradios only on Jessie

Reply:
Thanks for pointing this out. Created  #18